src/sage/repl/ipython_extension.py

```diff
@@ -357,12 +373,23 @@ def cython(self, line, cell):
           - ``--use-cache``
           - ``--create-local-c-file``
           - ``--annotate``
+          - ``--view-annotate``
           - ``--sage-namespace``
           - ``--create-local-so-file``
           - ``--no-compile-message``, ``--no-use-cache``, etc.
 
           See :func:`~sage.misc.cython.cython` for details.
 
+          For ``--view-annotate``, the following additional choices are allowed:
+
+          - ``--view-annotate=none`` (default)
+          - ``--view-annotate=auto`` (same as ``--view-annotate``); select one of the
```

Comment:
I think this default `none` is useless.

Reply:
and "auto" should be the default (for the case ``--view-annotate``).

Reply:
I interpret default to be "which option is chosen if nothing is specified", in that case the default is indeed it is not viewed.

Indeed user would never need to explicitly pass `--view-annotate=none` unless to override a previous `--view-annotate` option.

Reply:
What do you mean by "nothing is specified":
```
%%cyhon --view-annotate
```
or 
```
%%cyhon 
```
? 

Perhaps you mean the latter. But in the context, I think the former is understood. Then the default is "auto".

Reply:
I decided to simplify the documentation. The `=none` and `=auto` option is now undocumented (the user doesn't need to know that anyway)